eegrasp/eegrasp.py

```diff
@@ -46,13 +55,14 @@ def euc_dist(self, pos):
 
         Parameters
         ----------
-        pos -> 2d or 3d array of channels by dimensions
+        pos : ndarray.
+            2d or 3d array of channels by dimensions
 
         Returns
         -------
-        output: 2d array of channels by channels with the euclidean distance.
-        description: compute the euclidean distance between every channel in
-        the array
+        output: ndarray.
+            Dimension of the array is Channels by Channels with the euclidean distance between
```

Comment:
by Channels you mean number of channels?

Reply:
yes

Reply:
Then say so. Even better, use something linked to variables relevant to the code, like `len(something)` by `len(something)`